### DISCUSSION TITLE
App doesn't open on my Windows 11.

### DISCUSSION THREAD

**Discussion by EnzoHub2024** [ntsc-rs/ntsc-rs#163] [General]
Guys, i downloaded the standalone version of NTSC-RS, and suddently, while i opened it, it doesn't work! can you fix it for me pls

**Comment by ghost** [comment]
It should open

**Comment by ghost** [comment]
What version of Windows and build use?

**Comment by Degamisu** [comment]
This is an old post, @EnzoHub2024 may have figured it out, but here are a few things to note:

## Endless loading

Firstly, there is an issue with how the app starts. It happens to anyone. The issue can be described as an endless loading screen. This has been brought up many times, and I've found that you must wait `5-10 minutes` for it to properly load.

## Unsupported chipset/hardware

Another issue that has been brought up is outdated hardware. @valadaptive has mentioned a `software fallback renderer`* that can solve this issue, but so far this is not fixable other than getting more recent hardware.

>[!TIP]
>To make it easier for others to debug and pinpoint the error, you should provide at least some info on what happens. Does is open? If so, it would be useful to send a `video`!

\*`Software renderer` mentioned in #147
